assets/stylesheets/babble.css

```diff
@@ -84,3 +84,8 @@ li.babble-post .babble-post-content a.mention {
   padding: 5px;
   margin: 0;
 }
+
+.d-header .icons .unread-notifications.babble-unread-notifications {
```

Comment:
Oh crap… I definitively need to stop using that web client for more than one-liners. Will get back to that asap…